### ISSUE TITLE
Unable to create indexes: KeyError: 'name'

### ISSUE THREAD

**Issue by zopyx** [joowani/python-arango#117]
python-arango 5.2 when calling `add_persistent_index()`:
```
  File "bin/import-jsondump-into-arangodb", line 307, in <module>
    sys.exit(ugent.plone5migration.migration.import_jsondump_into_arangodb.main())
  File "/home/ajung/sandboxes/ugent-portaal-plone-4x/src/ugent.plone5migration/src/ugent/plone5migration/migration/import_jsondump_into_arangodb.py", line 67, in main
    collection.add_persistent_index(fields=["_paths_all[*]"], name="foo")
  File "/home/ajung/sandboxes/ugent-portaal-plone-4x/eggs/python_arango-5.2.0-py3.7.egg/arango/collection.py", line 1301, in add_persistent_index
    return self._add_index(data)
  File "/home/ajung/sandboxes/ugent-portaal-plone-4x/eggs/python_arango-5.2.0-py3.7.egg/arango/collection.py", line 1114, in _add_index
    return self._execute(request, response_handler)
  File "/home/ajung/sandboxes/ugent-portaal-plone-4x/eggs/python_arango-5.2.0-py3.7.egg/arango/api.py", line 66, in _execute
    return self._executor.execute(request, response_handler)
  File "/home/ajung/sandboxes/ugent-portaal-plone-4x/eggs/python_arango-5.2.0-py3.7.egg/arango/executor.py", line 82, in execute
    return response_handler(resp)
  File "/home/ajung/sandboxes/ugent-portaal-plone-4x/eggs/python_arango-5.2.0-py3.7.egg/arango/collection.py", line 1112, in response_handler
    return format_index(resp.body)
  File "/home/ajung/sandboxes/ugent-portaal-plone-4x/eggs/python_arango-5.2.0-py3.7.egg/arango/formatter.py", line 15, in format_index
    'name': body['name'],
KeyError: 'name'
```

**Comment by joowani** [comment]
Hi @zopyx,

Thanks for the bug report. What version of ArangoDB are you using?

**Comment by joowani** [comment]
Hi @zopyx,

I just released the fix in version 5.2.1.
